develop/migration_mt_django.rst

```diff
@@ -178,47 +525,45 @@ We need one final migration to tell Citus to mark tables for distribution. Creat
 
     operations = [
       migrations.RunSQL(
-        "SELECT create_distributed_table('mtdjango_store','id')"
+        "SELECT create_distributed_table('myapp_account','id')"
       ),
       migrations.RunSQL(
-        "SELECT create_distributed_table('mtdjango_product','store_id')"
+        "SELECT create_distributed_table('myapp_manager','account_id')"
       ),
       migrations.RunSQL(
-        "SELECT create_distributed_table('mtdjango_purchase','store_id')"
+        "SELECT create_distributed_table('myapp_project','account_id')"
+      ),
+      migrations.RunSQL(
+        "SELECT create_distributed_table('myapp_projectmanager','account_id')"
+      ),
+      migrations.RunSQL(
+        "SELECT create_distributed_table('myapp_task','account_id')"
       ),
-    ]
-
-With all the migrations created from the steps so far, apply them to the database with ``./manage.py migrate``.
-
-There's one more little detail. Server-side cursors do not work well with Citus. Go back to the database configuration in `settings.py` and include the following parameter:
```

Comment:
Does Citus now support cursors? We added this note about DISABLE_SERVER_SIDE_CURSORS because someone opened a github issue about it.

Reply:
Ah, not sure, I deleted this part because in my tests (in the unit tests of the django-multitenant library) I didn't have to change that.